src/pages/ReimbursementAccount/BankInfo/substeps/Plaid.tsx

```diff
@@ -0,0 +1,117 @@
+import {useIsFocused} from '@react-navigation/native';
+import React, {useCallback, useEffect} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx/lib/types';
+import AddPlaidBankAccount from '@components/AddPlaidBankAccount';
+import FormProvider from '@components/Form/FormProvider';
+import useLocalize from '@hooks/useLocalize';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as BankAccounts from '@userActions/BankAccounts';
+import * as ReimbursementAccount from '@userActions/ReimbursementAccount';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type * as OnyxTypes from '@src/types/onyx';
+import type {Errors} from '@src/types/onyx/OnyxCommon';
+
+type PlaidOnyxProps = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: OnyxEntry<OnyxTypes.ReimbursementAccount>;
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: OnyxEntry<OnyxTypes.ReimbursementAccountDraft>;
+
+    /** Contains plaid data */
+    plaidData: OnyxEntry<OnyxTypes.PlaidData>;
+};
+
+type PlaidProps = PlaidOnyxProps & SubStepProps;
+
+type ValuesToValidate = {
+    acceptTerms: boolean;
+};
+
+const bankInfoStepKeys = CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY;
```

Comment:
NAB: define it in one place and reuse (CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY is extracted multiple times like this)

Reply:
I think it kind of beats the purpouse. Its repeated maybe across 3 files and was introduced to shorten the notation and do not repeat `CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY` multiple times, IMO it can stay how it is

Reply:
I agree with @MrMuzyk